docs/docs/usage/getting-started.md

```diff
@@ -1,93 +1,194 @@
 ---
```

Comment:
I don’t think we need to host that page in the dotty repo. I think it makes sense to host here the documentation for language features, which are tightly coupled with the implementation in the compiler itself (so that we update both the documentation and the implementation in a single repository), and for pages that contain code examples (that can be checked by scaladoc).

Reply:
I guess we should not maintain all these 3 `usage` files and 1 `resources` (talks) in the dotty repo as well, so the only `reference` files will be preprocessed